lacci/lib/shoes.rb

```diff
@@ -42,6 +42,8 @@ module Kernel
 # small visual applications in Ruby.
 #
 module Shoes
+  include Shoes::Constants
```

Comment:
added this here because in some examples we use constants as Shoes::PI and in some we just say PI